rnd/autogpt_builder/src/components/CredentialsProvider.tsx

```diff
@@ -0,0 +1,85 @@
+import AutoGPTServerAPI, { CredentialsMetaResponse } from "@/lib/autogpt-server-api";
```

Comment:
At this point probably make a providers folder

Reply:
I agree and more - I think we need a dir for Flow/Agent components and maybe more. But let's do it in another PR.